src/main/kotlin/no/nav/syfo/App.kt

```diff
@@ -74,6 +75,11 @@ fun main() {
             kafkaAivenProducerConfig<KafkaArbeidstakervarselSerializer>(kafkaEnvironment = environment.kafka)
         )
     )
+    val expiredForhandsvarselProducer = ExpiredForhandsvarselProducer(
+        producer = KafkaProducer(
+            kafkaAivenProducerConfig<KafkaArbeidstakervarselSerializer>(kafkaEnvironment = environment.kafka)
```

Comment:
Må bruke riktig `Serializer` her (som stemmer med record-value for produceren). Egentlig litt rart at det ikke feiler compile-time :thinking: